lib/remote/actionshandler.cpp

```diff
@@ -100,12 +100,21 @@ bool ActionsHandler::HandleRequest(
 	}
 
 	int statusCode = 500;
+	std::set<int> statusCodes;
 
 	for (const Dictionary::Ptr& res : results) {
-		if (res->Contains("code") && res->Get("code") >= 200 && res->Get("code") <= 299) {
-			statusCode = 200;
-			break;
+		if (!res->Contains("code")) {
```

Comment:
Now that the body of this loop got quite short, this early continue is of little use. I'd just invert the condition and put the insert within the if.